### PR TITLE
Faster get_Video_No_Watermark_ID

### PR DESCRIPTION
**Pull request by rezaarifandee**
TL;DR; Downloading only part of video with "vid:" info instead of downloading the whole file to speed up the function.

I found that the function is terribly slow, after profiling it, it seems that it spend most of the time to download the video with watermark. After analyzing around 800 of videos, i found that the "vid:" info is ALWAYS placed between byte 5000-25000 and there is no need to download the whole file for it. With adding Ranges header, we can download only the specific part of the file, improving the speed. However i am not sure if there any videos with more range bigger than that, so i configured the range to 1000-30000 just to be sure.

### CODE REVIEW

**Comment by rezaarifandee** [comment]
Oh, btw, the function went from on average 15 seconds per video to 1.5 seconds per video.

**Comment by davidteather** [comment]
Thanks! You really sped it up a lot. It'll be released along with a few other speed fixes later today.

**Comment by rezaarifandee** [comment]
No, thank YOU! for creating and maintaining this.
